examples/README.md

```diff
@@ -429,9 +429,12 @@ on IBM Bluemix, follow the instructions below. If you are not a Bluemix user, yo
 1. Login to Bluemix and initialize the container environment using ```bluemix login``` and ```bluemix ic init```
 
 1. Create Bluemix routes (DNS names) for the registry, controller and the bookinfo app's gateway:  
-    ```cf create-route myspace mybluemix.net -n mya8-registry```
-    ```cf create-route myspace mybluemix.net -n mya8-controller```
-    ```cf create-route myspace mybluemix.net -n mya8-bookinfo```
+    ```bash
+cf create-route <your bluemix space> mybluemix.net -n <your registry route>
+cf create-route <your bluemix space> mybluemix.net -n <your controller route>
+cf create-route <your bluemix space> mybluemix.net -n <your bookinfo route>
+```
+where `<your bluemix space>` is the name of your Bluemix space and `<your route ...>` is a unique route name for `registry`, `controller` and `bookinfo`. For example `my-space-name-a8-registry`, `my-space-name-a8-controller`, etc. Make a note of the route names you choose for the next step.
     
 
 1. Customize the [bluemixrc](bluemix/.bluemixrc) file in the following manner:
```

Comment:
may be say 

[examples/bluemix/.bluemixrc](bluemix/.bluemixrc) file in the ..

?